GOVERNANCE.md

```diff
@@ -1,25 +1,100 @@
+* [Carvel-vendir](https://github.com/vmware-tanzu/carvel-vendir): Declaratively state what files should be in a directory
+
+**Experimental:**
+* [kwt](https://github.com/vmware-tanzu/carvel-kwt)
+* [terraform-provider-carvel](https://github.com/vmware-tanzu/terraform-provider-carvel)
+* [carvel-secretgen-controller](https://github.com/vmware-tanzu/carvel-secretgen-controller)
+
+**Installation:**
+* [homebrew-carvel](https://github.com/vmware-tanzu/homebrew-carvel)
+* [carvel-docker-image](https://github.com/vmware-tanzu/carvel-docker-image)
+* [asdf-carvel](https://github.com/vmware-tanzu/asdf-carvel)
+* [carvel-setup-action](https://github.com/vmware-tanzu/carvel-setup-action)
+
+**Plugins:**
+* [ytt.vim](https://github.com/vmware-tanzu/ytt.vim)
+
+**Examples:**
+* [carvel-simple-app-on-kubernetes](https://github.com/vmware-tanzu/carvel-simple-app-on-kubernetes)
+* [carvel-ytt-library-for-kubernetes](https://github.com/vmware-tanzu/carvel-ytt-library-for-kubernetes)
+* [carvel-ytt-library-for-kubernetes-demo](https://github.com/vmware-tanzu/carvel-ytt-library-for-kubernetes-demo)
+* [carvel-guestbook-example-on-kubernetes](https://github.com/vmware-tanzu/carvel-ytt-library-for-kubernetes-demo)
+
+---
+# Community Roles
+* **Users:** Anyone that uses a tool within Carvel.
+* **Contributors:** A contributor is anyone that contributes to one or more projects (documentation, code reviews, responding to issues, participation in proposal discussions, contributing code, etc.) or is continuously active in the Carvel community.
+* **Maintainers:** The Carvel project leaders. They are responsible for the overall health and direction of the project and responsible for releases. Maintainers are responsible for one or more components within the project. Some maintainers act as a technical lead for specific components. Carvel maintainers are broken down into three sub-roles: maintainer, reviewer, approver. 
+    * **Maintainer:** Maintainers are expected to contribute code and documentation, triage issues, proactively fix bugs, and perform maintenance tasks for these components.
+    * **Reviewer:** They have all the responsibilities of a maintainer with additional responsibilities and permissions. A reviewer can review code for quality and correctness on a tool. They are knowledgeable about both the codebase and software engineering principles. They can approve pprovers' contributions.
+    * **Approver:** They have all the responsibilities of a eviewer with additional responsibilities and permissions. An Approver can both review and approve code contributions from anyone. While code review is focused on code quality and correctness, approval is focused on holistic acceptance of a contribution including backward/forwards compatibility, adhering to API and flag conventions, subtle performance and correctness issues, interactions with other parts of the system, etc.
```

Comment:
`eviewer` is missing an `r`

Reply:
For consistency, `An Approver` doesn't need to be capitalized and could be `An approver`.